File: SAcubeCO.cpp
#include<iostream>
using namespace std;

class SurfaceArea{
    public: 
        int area;
        SurfaceArea(int a){
            area= 4*a*a;
        }
        SurfaceArea(int a, int b){
            area= (4*a*a)+(4*b*b);
        }
        void calculateArea(){
            cout<<area<< endl;
        }
};

int main(){
    SurfaceArea o(0);
    SurfaceArea o1(0,0);
    int choice, a, b;
    cout<<"Enter number of cubes: ";
    cin>>choice;
    switch (choice)
    {
    case 1:
        cout<<"Enter side of cube 1: ";
        cin>>a;
        o = SurfaceArea(a);
        o.calculateArea();
        break;
    
    case 2:
        cout<<"Enter side of cube 1: ";
        cin>>a;
        cout<<"Enter side of cube 2: ";
        cin>>b;
        o1 = SurfaceArea(a,b);
        o1.calculateArea();
        break;
    
    default:
        break;
    }
    return 0;
}

File: SAcubeCO.java
import java.util.Scanner;
class SAcubeCO{
    private int a, b;
    public int area;
    SAcubeCO(int a){
        area = 4*a*a;
    }
    SAcubeCO(int a, int b){
        area = (4*a*a)+(4*b*b);
    }
    void calculateArea(){
        System.out.println(area);
    }

    public static void main(String[] args) {
        int n1, side1, side2;
        Scanner sc= new Scanner(System.in);
        System.out.println("Enter number of cube: ");
        n1= sc.nextInt();
        switch (n1) {
            case 1:
                System.out.println("Enter side: ");
                side1 = sc.nextInt();
                SAcubeCO obj= new SAcubeCO(side1);
                obj.calculateArea();
                break;
            case 2:
                System.out.println("Enter side of cube 1: ");
                side1 = sc.nextInt();
                System.out.println("Enter side of cube 2: ");
                side2 = sc.nextInt();
                SAcubeCO obj1= new SAcubeCO(side1,side2);
                obj1.calculateArea();
                break;

            default:
                break;
        }
        sc.close();
    }
}